addons/medical_damage/functions/fnc_getArmor.sqf

```diff
@@ -0,0 +1,73 @@
+#include "script_component.hpp"
+/*
+ * Author: Pterolatypus
+ * Checks a unit's equipment to calculate the total armor on a hitpoint.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Array of hitpoint armor <ARRAY>
+ *
+ * Example:
+ * [player] call ace_medical_damage_fnc_getArmor
+ *
+ * Public: No
+ */
+
+params ["_unit"];
+
+private _uniform = uniform _unit;
+// If unit is naked, use its underwear class instead
+if (_uniform isEqualTo "") then {
+    _uniform = getText (configFile >> "CfgVehicles" >> typeOf _unit >> "nakedUniform");
+};
+
+private _gear = [
+    _uniform,
+    vest _unit,
+    headgear _unit
+];
+
+private _rags = _gear joinString "$";
+private _var = format [QGVAR(armorCache%1), _rags];
+(localNamespace getVariable [_var, [""]]) params ["_prevRags", "_armor"];
```

Comment:
Why localNamespace?

Reply:
Why not? 
Reduced size of save game at least.

Reply:
Because it reduces readability and comprehensibility of the code. Save game size is nothing you should be concerned about.

Reply:
`localNamespace` is intended for caching